ergo-chain-sync/Cargo.toml

```diff
@@ -11,8 +11,18 @@ futures = "0.3.25"
 isahc = { version = "1.7.2", features = ["json"] }
 serde = { version = "1.0.147", features = ["derive"] }
 serde_json = "1.0.88"
-ergo-lib = { version = "0.21.1", features = ["json"] }
+#ergo-lib = { version = "0.21.1", features = ["json"] }
+ergo-lib = { git = "https://github.com/ergoplatform/sigma-rust", branch = "i655_move_arbitrary_impls", features = ["json"] }
```

Comment:
I made a PR in sigma-rust https://github.com/ergoplatform/sigma-rust/pull/656 so that we can easily generate `BlockId`s and `Transaction`s in our tests below. Note that `cargo` doesn't allow us to mix and match dependency sources.